README.md

```diff
@@ -102,19 +85,19 @@ For more information on the importance of a professional README for your reposit
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Displays the available books]**
+- **[Enabling the addition of new students or teachers]**
+- **[Allowing the addition of new books]**
+- **[Storing records of book borrowings along with the corresponding dates]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- [Live Demo Link](Coming Soon)
 
```

Comment:
- [ ] Kindly make sure to remove the `Live Demo` section altogether from the README because this project will not deploy as a web app